projects/ion/src/lib/alert/alert.component.ts

```diff
@@ -35,4 +43,10 @@ export class IonAlertComponent implements OnInit {
   ngOnInit(): void {
     this.setIcon();
   }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.hideBackground && changes.hideBackground.currentValue) {
+      this.closable = false;
+    }
+  }
```

Comment:
I believe that you can add this rule in OnInit.

Reply:
I added the code to the ngOnInit() method, but when I click on the hideBackground button in the Storybook, the closable property remains true.